### DISCUSSION TITLE
Pre-requisites to install MISP on RHEL 8.X

### DISCUSSION THREAD

**Discussion by siddarthtalupula** [MISP/MISP#8529] [Q&A]
Hello,
I am looking for guidance in terms of the pre-requisites to install MISP on RHEL 8. X
I tried to install but there were too many errors during the installation. The installation is completed, however, when I try to launch the portal, it tells SERVICE UNAVAILABLE.
I am not sure if Apache is installed by default.

Thanks in advance.
Siddarth

**Comment by lukaszs1942** [comment]
Hi,

I have similar issue, is there anyone who sucesfully installed MISP on RH8? In my opinion there is an issue with CakePHP module.

OK i digged a bit more and found out remi repo was failing test so whole php section didnt work and was not installed. After running rpm --rebuilddb it started  to work.

BR

**Reply by siddarthtalupula** [reply]
@lukaszs1942 , did you manage to successfully install MISP on RHEL 8?

Thanks in advance.
Siddarth

**Reply by lukaszs1942** [reply]
Yes but only at home in my private environment.

**Reply by siddarthtalupula** [reply]
@lukaszs1942 , can you please share your workaround. How did you fixthe issue.

Thanks in advance
Siddarth

**Reply by lukaszs1942** [reply]
After installation need to update submodules and install cakereque + set correct permissions

dnf install https://rpms.remirepo.net/enterprise/remi-release-8.rpm

sudo -u apache php composer.phar require kamisama/cake-resque:4.1.2
sudo -u apache php composer.phar config vendor-dir Vendor
sudo -u apache php composer.phar install

submodules
git submodule init
git submodule update
